lib/game.rb

```diff
@@ -53,61 +53,61 @@ def end_game
   def place_player_ship(ship_parameter)
     puts "Enter the squares for the #{ship_parameter.name} (#{ship_parameter.length} spaces):"
     response = gets.chomp.upcase.gsub(",", " ").split(" ")
-    until @player_data["player_board"].place(ship_parameter, response) != nil do
+    until @player_data[:player_board].place(ship_parameter, response) != nil do
       puts "Those are invalid coordinates. Please try again:"
       response = gets.chomp.upcase.gsub(",", " ").split(" ")
     end
-    puts @player_data["player_board"].render(true)
+    puts @player_data[:player_board].render(true)
   end
 
   def place_computer_ship(ship_parameter)
     ship_coordinates = nil
     until ship_coordinates != nil do
-      random_coordinates = @computer_data["computer_board"].cells.keys.sample(ship_parameter.length)
-      ship_coordinates = @computer_data["computer_board"].place(ship_parameter, random_coordinates)
+      random_coordinates = @computer_data[:computer_board].cells.keys.sample(ship_parameter.length)
+      ship_coordinates = @computer_data[:computer_board].place(ship_parameter, random_coordinates)
     end
   end
 
   def display_boards
     puts "=============COMPUTER BOARD============="
-    puts @computer_data["computer_board"].render
+    puts @computer_data[:computer_board].render
     puts "=============PLAYER BOARD==============="
-    puts @player_data["player_board"].render(true)
+    puts @player_data[:player_board].render(true)
   end
 
   def player_shot
     puts "Enter the coordinate for your shot:"
-    target = gets.chomp.upcase.delete(" ")[0..1]
+    target_parameter = gets.chomp.upcase.delete(" ")[0..1]
```

Comment:
do you think we should have a helper method for the two types of user inputs (single coordinate, and multiple coordinates) to "fix" the format? Just a thought.

Reply:
Since we use it a few times